server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/service/UserService.java

```diff
@@ -60,21 +61,23 @@ public void removeUser(Username username) throws UsersRepositoryException {
         usersRepository.removeUser(username);
     }
 
-    public void upsertUser(Username username, char[] password) throws UsersRepositoryException {
+    public void upsertUser(Username username, char[] password, Boolean isForced) throws UsersRepositoryException, UserConflictException {
```

Comment:
boolean parameter in a method is a code smell showing we actualy need 2 methods.

How about `void upsertUser(Username username, char[] password)` & `void insertUser(Username username, char[] password)` ?